src/components/Storyblok/SbThemeCard.tsx

```diff
@@ -5,24 +5,23 @@ import { SbVerticalCardProps } from './SbVerticalCard';
 import { AccentBgColorType } from '../../utilities/datasource';
 import { paletteAccentColors } from '../../utilities/colorPalettePlugin';
 
-type SbThemeCardProps = Omit<SbVerticalCardProps, 'isSmallHeading'>;
+export type SbThemeCardProps = Omit<SbVerticalCardProps, 'isSmallHeading'>;
 
 export const SbThemeCard = ({
   blok: {
     _uid,
     heading,
     headingLevel,
     body,
-    // TODO: seperate alt as separate field
-    image: { filename, focus, alt } = {},
```

Comment:
I don't think there's a need for alt text for the theme card other than the empty ""